site/src/pages/TemplatePage/TemplateVersionsPage/VersionRow.tsx

```diff
@@ -39,7 +39,7 @@ export const VersionRow: React.FC<VersionRowProps> = ({
     <TimelineEntry
       data-testid={`version-${version.id}`}
       {...clickableProps}
-      css={[styles.row, isActive && styles.active]}
+      css={[styles.row]}
```

Comment:
Why did we remove the active styling here?

Reply:
because the active version already has a "Active" badge on it, so it's kind of weird for it to also be a different background color. it's not really clear that that's _why_ the background color is different. we should reserve that sort of styling for keyboard navigation and such.